tests/services/data.service.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { of } from 'rxjs';
+import { DataService } from '../../src/app/core/services/data.service';
+import { GithubService } from '../../src/app/core/services/github.service';
+import { autoSpy } from '../auto-spy';
+import { csvString, jsonData, dataFileTeamStructure } from '../constants/data.constants';
+
+describe('DataService', () => {
+  describe('getDataFile', () => {
+    it('should return the correct data', done => {

```

Comment:
@geshuming, minor nit picks. Since we are standardising to this code style, should we:
1. Write the function name including the parameters.
2. The "it" statement should be a condition for the test case.

Reply:
I think we can leave this nit pick for a separate PR since this PR has been ongoing for quite a while. 
I will create an issue tracker for this.